.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
             - run: echo $PYVERSION > ver.txt
 
             - restore_cache:
-                key: deps1-{{ .Branch }}-{{ checksum "tests/requirements.txt" }}-{{ checksum "ver.txt" }}
+                key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "tests/requirements.txt" }}-{{ checksum "ver.txt" }}
```

Comment:
The dash version is in `requirements.txt`, updating this file during a PR causes the cache not to be updated and the changes not to be reflected in the build environment. Adding.